packages/camera/camera/lib/src/camera_preview.dart

```diff
@@ -41,26 +40,25 @@ class CameraPreview extends StatelessWidget {
         : Container();
   }
 
-  DeviceOrientation _getApplicableOrientation() {
-    return controller.value.isRecordingVideo
-        ? controller.value.recordingOrientation!
-        : (controller.value.lockedCaptureOrientation ??
-            controller.value.deviceOrientation);
-  }
-
   bool _isLandscape() {
     return [DeviceOrientation.landscapeLeft, DeviceOrientation.landscapeRight]
         .contains(_getApplicableOrientation());
   }
 
   int _getQuarterTurns() {
-    int platformOffset = defaultTargetPlatform == TargetPlatform.iOS ? 1 : 0;
     Map<DeviceOrientation, int> turns = {
       DeviceOrientation.portraitUp: 0,
       DeviceOrientation.landscapeLeft: 1,
       DeviceOrientation.portraitDown: 2,
       DeviceOrientation.landscapeRight: 3,
     };
-    return turns[_getApplicableOrientation()]! + platformOffset;
+    return turns[_getApplicableOrientation()]!;
+  }
+
+  DeviceOrientation _getApplicableOrientation() {
```

Comment:
Just to confirm, there is no diff in this method right? The method was just moved down?

Reply:
Correct, it made more sense to me to keep it closer to where it was used. I see now that it causes a but of noice

Reply:
keeping it close to where it was used makes sense to me!